src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -33,7 +33,7 @@ public void Initialize(EntityInfo entity, Func<IncomingMessageDetails, ReceiveCo
             receiveMode = settings.Get<ReceiveMode>(WellKnownConfigurationKeys.Connectivity.MessageReceivers.ReceiveMode);
 
             incomingCallback = callback;
-            this.errorCallback = errorCallback;
+            this.errorCallback = errorCallback ?? EmptyErrorCallback;
```

Comment:
It is not entirely true. The interface is public. It is a one to one relationship of interface to class. The interface is destined to be used in the topology operator which has a public method outside the normal lifecycle of the class called OnError which is on the interface as well. When you don't call this method then the func is always null. Before we correct that design it requires a null check